05-Jason-Luo/wk01/5-fri/trainsystem.js

```diff
@@ -0,0 +1,102 @@
+// There are 3 subway lines:
+// The N line has the following stops: Times Square, 34th, 28th, 23rd, Union Square, and 8th
+// The L line has the following stops: 8th, 6th, Union Square, 3rd, and 1st
+// The 6 line has the following stops: Grand Central, 33rd, 28th, 23rd, Union Square, and Astor Place.
+// All 3 subway lines intersect at Union Square, but there are no other intersection points. (For example, this means the 28th stop on the N line is different than the 28th street stop on the 6 line, so you'll have to differentiate this when you name your stops in the arrays.)
+// Tell the user the number of stops AND the stops IN ORDER that they will pass through or change at.
+
+const trainlines = {
+    N: ["Times Square", "34th", "N-28th", "N-23rd", "Union Square", "N-8th"],
+    L: ["L-8th", "6th", "Union Square", "3rd", "1st"],
+    6: ["Grand Central", "33rd", "6-28th", "6-23rd", "Union Square", "Astor Place"],
+};
+
+function stationToUnion(selectLine, selectUnion, selectStation) {
+    if (selectStation > selectUnion) {
```

Comment:
This `if else` statement is repeated in your `.stationToUnion()`, `.firstTrip()` & `secondTrip()` functions.
Remember the _DRY_ principle (Don't Repeat Yourself). I would think about how you could consolidate this conditional logic in one place.